token/core/fabtoken/v1/setup/setup.go

```diff
@@ -167,8 +169,13 @@ func (p *PublicParams) Deserialize(raw []byte) error {
 	if err != nil {
 		return errors.Wrapf(err, "failed to deserialize public parameters")
 	}
-	if container.Identifier != p.Label {
-		return errors.Errorf("invalid identifier, expecting 'fabtoken', got [%s]", container.Identifier)
+	expectedID := string(core.TokenDriverName(FabtokenIdentifier, ProtocolV1))
```

Comment:
for disambiguation I would call them `TokenDriverIdentifier` (instead of name), `TokenDriveryType` (instead of identifier), and `TokenDriverVersion`